### ISSUE TITLE
Bad hint in UserWarning in `markdown.py`

### ISSUE THREAD

**Issue by kamichal** [pypa/readme_renderer#150]
If readme-renderer[md] is not available such a confusing information is being printed:
```
/local/lib/python2.7/site-packages/readme_renderer/markdown.py:38: UserWarning: Markdown renderers are not available. Install 'readme_render[md]' to enable Markdown rendering.
  warnings.warn(_EXTRA_WARNING)
```
But there is no such package `readme_render[md]` in pypi:
`No matching distribution found for readme_render[md]`
It contains two typos and should be corrected to `readme-renderer[md]`.

**Comment by di** [comment]
This was fixed in #140, but we need to make a new release. I'll try to do this today.

**Comment by kamichal** [comment]
OK, nice, I missed it.
